05/app.js

```diff
@@ -41,7 +41,10 @@ function insertUsers(usersList) {
 function postUser(evt) {
   evt.preventDefault();
   const {nameData} = bundleUserInput();
-  console.log(JSON.stringify(nameData));
+  const {firstName, lastName} = nameData;
+  if (!firstName || !lastName) {
+    return console.log('Invalid data');
```

Comment:
W takim wypadku zapisałbym tak:
```
if (!firstName || !lastName) {
    console.log('Invalid data'); // wyświetlam komunikat ps. Pamiętaj, że żaden użytkownik nie patrzy do konsoli :P
    return; // zatrzymuje działanie kodu
```

Niby to samo, ale lepiej pokazuje intencje tego co robimy; `return console.log('...')` zwróci `undefined` i może to być zrozumiałe jako błędne wykorzystanie return itp.